Documentation for "ask the community" feature (#3804)
mattermost/docs: source/administration/config-settings.rst

| This feature's ``config.json`` setting is ``"CustomBrandText": ""`` with string input.                                                                               |
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+

Enable Ask Community Link
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> When true, "Ask the community" link appears on the Mattermost user interface and Main Menu, which allows users to join the Mattermost Community to ask questions and help others troubleshoot issues.

Reviewer: ```suggestion
**True**: **Ask the community** link is visible on the Mattermost user interface and Main Menu. When selected, users can join the Mattermost Community to ask questions and help others troubleshoot issues.
```

> When false, the link is hidden from users.

Reviewer: ```suggestion
**False**: The link is not visible to users.
```

> **True**: **Ask the community** link is visible in the Mattermost channel header, under the **Help** menu. When clicked, users are redirected to https://mattermost.com/pl/default-ask-mattermost-community/, where they can join the Mattermost Community to ask questions and help others troubleshoot issues.

Author: ```suggestion
**True**: **Ask the community** link is visible in the Mattermost channel header, under the **Help** menu. When clicked, users are redirected to https://mattermost.com/pl/default-ask-mattermost-community/, where they can join the Mattermost Community to ask questions and help others troubleshoot issues. This option is not available on the mobile apps.
```

+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+

> | This feature's ``config.json`` setting is ``"enable_ask_community_link": ""`` with options ``true`` and ``false``.                                                   |

Reviewer 2: value is true by default?
